format/postgres/pg_wal.go

```diff
@@ -0,0 +1,98 @@
+package postgres
+
+import (
+	"fmt"
+
+	"github.com/wader/fq/format/postgres/common/pg_wal/pgproee"
+	"github.com/wader/fq/format/postgres/common/pg_wal/postgres"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+
+	"strconv"
+	"strings"
+)
+
+// partial parsing of WAL
+
+func init() {
+	interp.RegisterFormat(decode.Format{
+		Name:        format.PG_WAL,
+		Description: "PostgreSQL write-ahead log file",
+		DecodeFn:    decodePGWAL,
+		DecodeInArg: format.PostgresWalIn{
+			Flavour: PG_FLAVOUR_POSTGRES14,
+			Lsn:     "",
```

Comment:
Maybe an example how to use this?  `fq -d pg_wal -o lsn='"123/456"' ...` something?